superset-frontend/src/explore/components/OptionControls.tsx

```diff
@@ -92,7 +92,26 @@ export const LabelsContainer = styled.div`
   border-radius: ${({ theme }) => theme.gridUnit}px;
 `;
 
-export const AddControlLabel = styled.div`
+export const DndLabelsContainer = styled.div<{
+  canDrop?: boolean;
+  isOver?: boolean;
+}>`
+  padding: ${({ theme }) => theme.gridUnit}px;
+  border: ${({ canDrop, isOver, theme }) => {
+    if (isOver && canDrop) {
+      return `dashed 1px ${theme.colors.info.dark1}`;
+    }
+    if (isOver && !canDrop) {
+      return `dashed 1px ${theme.colors.error.dark1}`;
+    }
```

Comment:
It feels like `dashed 1px` doesn't stick out quite enough. IMO `2px` looked better, but then the default `1 px` causes this to resize when hovering in/out, so that doesn't work quite so well without addditional tuning of the default borders.

Reply:
I think that we should also add an indicator for `canDrop && !isOver` so that user knows where he can drag a column

Reply:
> I think that we should also add an indicator for `canDrop && !isOver` so that user knows where he can drag a column

Yeah I like that idea 👍